### PR TITLE
Added KMS key for encryption of remote files

### PR DESCRIPTION
**Pull request by arjunrn**
Added KMS key for encryption of remote files. The E2E test for Ubuntu will fail because of the change in format of the `files.yaml`

### CODE REVIEW

File: cluster/node-pools/master-ubuntu-default/userdata.yaml [hunk 1]
```diff
@@ -21,6 +21,7 @@ write_files:
     path: /var/run/s3-certs.env
     content: |
       S3_CERTS_BUCKET={{ .S3GeneratedFilesPath }}
+      AWS_DEFAULT_REGION=eu-central-1
```
Comment (mikkeloscar) [inline cluster/node-pools/master-ubuntu-default/userdata.yaml]: Should be `{{ .Cluster.Region }}`

File: cluster/node-pools/worker-ubuntu-default/userdata.yaml [hunk 2]
```diff
@@ -13,6 +13,7 @@ write_files:
     path: /var/run/s3-certs.env
     content: |
       S3_CERTS_BUCKET={{ .S3GeneratedFilesPath }}
+      AWS_DEFAULT_REGION=eu-central-1
```
Comment (mikkeloscar) [inline cluster/node-pools/worker-ubuntu-default/userdata.yaml]: Should be `{{ .Cluster.Region }}`

**Comment by arjunrn** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**